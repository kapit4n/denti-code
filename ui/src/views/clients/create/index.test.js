import { render, screen, fireEvent } from '@testing-library/react';
import Create from './index';

test('renders react name inputs', () => {
  const screen = render(<Create />);

  const nameInput = screen.queryByPlaceholderText(/First Name/i);
  expect(nameInput).toBeTruthy();

  expect(nameInput.value).toBe("")
  fireEvent.change(nameInput, { target: { value: "Luis" } })
  expect(nameInput.value).toBe("Luis")

  const lastName = screen.queryByPlaceholderText(/Last Name/i);
  expect(lastName).toBeTruthy();
  expect(lastName.value).toBe("")
  fireEvent.change(lastName, { target: { value: "Arce" } })
  expect(lastName.value).toBe("Arce")

  const submitBtn = screen.queryByText(/Submit/i);
  expect(submitBtn).toBeTruthy();

  fireEvent.click(submitBtn)

});

