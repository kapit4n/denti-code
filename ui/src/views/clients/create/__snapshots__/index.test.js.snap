// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `ShallowWrapper {}`;

exports[`renders 2`] = `ShallowWrapper {}`;
